jenkins_job_wrecker/modules/handlers.py

```diff
@@ -177,6 +177,10 @@ def childcustomworkspace(top, parent):
     parent.append(['child-workspace', top.text])
 
 
+def childcustomworkspace(top, parent):
```

Comment:
This shouldn't have been here as it's already in there. flake also gives errors.

Reply:
Good finding. Fixed.